includes/wp-cli/class-events.php

```diff
@@ -634,6 +639,27 @@ private function delete_completed_events( $args, $assoc_args ) {
 
 		\WP_CLI::success( __( 'Entries removed', 'automattic-cron-control' ) );
 	}
+
+	/**
+	 * Delete events that have a particular schedule
+	 *
+	 * @param array $args Array of positional arguments.
+	 * @param array $assoc_args Array of flags.
+	 */
+	private function delete_events_by_schedule( $args, $assoc_args ) {
+		if ( 'null' === strtolower( $assoc_args['schedule'] ) ) {
```

Comment:
I guess the `null` value is for events that are not recurring? Might be a good idea to have some inline docs making that clear

Reply:
Ya - also open to other terms for the name...`single`, `now`, `none`?